### PR TITLE
[SW-384] Multisig account address is missing on Propose and approve screen

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

File: src/popup/components/TransferReview.vue [hunk 1]
```diff
@@ -389,13 +398,22 @@ export default defineComponent({
...
+      activeMultisigAccount,
     };
   },
 });
 </script>
 
 <style scoped lang="scss">
-.details-item {
-  margin-top: 16px;
+.review-wrapper {
```
Comment (CedrikNikita) [inline src/popup/components/TransferReview.vue:408]: I suggest to rename the root class of the component to it's name, but in kebab-case as in every other place in the project.

File: src/popup/components/TransferReview.vue [hunk 2]
```diff
@@ -389,13 +398,22 @@ export default defineComponent({
...
+  .details-item {
+    margin-top: 16px;
+  }
+
+  .multisig-account {
+    display: flex;
+    justify-content: center;
+  }
 }
+
```
Comment (CedrikNikita) [inline src/popup/components/TransferReview.vue:418]:
```suggestion
```

File: src/popup/components/TransferReview.vue [hunk 3]
```diff
@@ -1,10 +1,17 @@
 <template>
-  <div class="review-wrapper">
+  <div class="transfer-review">
     <ModalHeader
       :title="isMultisig ? $t('modals.multisigTxProposal.title') : $t('pages.send.reviewtx')"
       :subtitle="isMultisig ? null : $t('pages.send.checkalert')"
     />
 
+    <div
+      v-if="isMultisig"
```
Comment (subhod-i) [inline src/popup/components/TransferReview.vue:9]: `v-show` instead of `v-if`

**Review by subhod-i — APPROVED** [review]

File: src/popup/components/TransferReview.vue (hunk 3 above)
Comment (konradodo) [reply src/popup/components/TransferReview.vue:9]: hmm i always prefer v-if if i not need render something

**Pull request merged**